screens/Profile/ExternalProfileScreen/ExternalProfileScreen.tsx

```diff
@@ -77,6 +78,7 @@ const ExternalProfileScreen = () => {
 
   return (
     <View style={styles.container}>
+      <BackArrow/>
```

Comment:
When testing with the app it doesn't render correctly. You should try putting it outside of the container. Moreover the position is absolute so take care about overlapping on layouts

Reply:
There is another PR opened to fix this problem (#243 )